src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Logging best practices guide
+tags:
+ - New Relic solutions
+ - Best practices guides
+ - Logs
+ - Logging
+translate:
+ - jp
+metaDescription: 'Best practices for using New Relic logs'
+redirects:
+---
+
+import parsingExample from 'images/logs-parsing-example.png'
+import image from 'images/image.png'
+
+Welcome to the New Relic logging best practices guide. Here you'll find recommendations for:
+
+- [Forwarding logs](#forwarding-logs)
+- [Data partitions](#partitions)
+- [Parsing logs](#parsing-logs)
+- [Drop filter rules](#drop-rules)
+- [Log sizing](#sizing)
+- [Searching logs](#searching-logs)
+- [What's next?](#whats-next)
+
+## Forwarding logs [#forwarding-logs]
+
+* When forwarding logs, always use our [New Relic infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent) and/or [APM agents](/docs/apm/new-relic-apm/getting-started/get-started-logs-context#agents) when possible. If you cannot use New Relic agents, use other supported agents (FluentBit, Fluentd, Logstash, etc.).
+
+Here are some Github example configurations for supported logging agents:
+   * [FluentBit examples](https://github.com/newrelic/fluentbit-examples) 
+   * [Fluentd examples](https://github.com/newrelic/fluentd-examples/)
+   * [Logstash examples](https://github.com/newrelic/logstash-examples)
+
+<Callout variant="warning">
+If your logs are stored in a directory on an underlying host/container and are instrumented by our infrastructure agent to collect logs, you may see duplicate logs collected by both the infrastructure agent and APM agent. You may choose to use APM logs, in which case you should edit the infrastructure agent configurations to stop collecting logs, or vice versa.
```

Comment:
@marcelschlapfer @lchapman4 I'm trying to determine which docs link to use, and looking at Line 37, it alternates between talking about the infra agent/apm agents and APM logs. It seems like it's combining infra and apm agents, and the customer should be using either the infra or apm agent to forward the logs. Do they need to [configure the infra agent configuration](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), or [enable/disable APM logs](https://docs.newrelic.com/docs/logs/logs-context/disable-automatic-logging/)?

Reply:
Yeah, this is trying to point out the possibility of getting duplicate logs if you are using both infra agent and APM agent to send application logs.  I guess this could be worded better ... also maybe we could point out the APM Logs Quickstart which would help them identify if there are duplicate logs.  @lchapman4 what are your thoughts?

Reply:
Agree with Marcel. I think we can link to the quickstart so customers can watch for duplicate logs. link here: https://newrelic.com/instant-observability/apm-logs

Reply:
I took this one step further and linked to [this doc](https://docs.newrelic.com/docs/logs/logs-context/upgrade-to-automatic-logs-context/), which includes our recommendations to avoid duplicate logging. Let me know if this makes sense in my new PR: https://github.com/newrelic/docs-website/pull/11400/files